modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPUtils.macro

```diff
@@ -16,6 +16,35 @@ definition {
 		}
 	}
 
+	macro assertDataOnEVPValue {
+		ScrollBy(value1 = "0, 1000");
```

Comment:
do we have another way to scroll until the specific element?
this way will only give a scroll without parameters.

Reply:
AssertVisible(  it would be ideal?

Reply:
```suggestion
		`ScrollWebElementIntoView(
		ScrollBy(value1 = "0, 1000");
		ScrollBy(value1 = "0, 1000");
			key_fieldValue = ${activityName},
			locator1 = "PRMMDFRequest#SUMMARY_FIELD");`
```